keyboards/setta21/rev1/rev1.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "setta21.h"
+
+//void promicro_bootloader_jmp(bool program);
+
+#define LAYOUT( \
```

Comment:
We do actually have community layout support for this layout:

```suggestion
#define LAYOUT_numpad_6x4( \
```